pyqtgraph/GraphicsScene/GraphicsScene.py

```diff
@@ -154,35 +158,40 @@ def mousePressEvent(self, ev):
                     break
         
     def mouseMoveEvent(self, ev):
-        self.sigMouseMoved.emit(ev.scenePos())
-        
-        ## First allow QGraphicsScene to deliver hoverEnter/Move/ExitEvents
-        QtGui.QGraphicsScene.mouseMoveEvent(self, ev)
-        
-        ## Next deliver our own HoverEvents
-        self.sendHoverEvents(ev)
-        
-        if int(ev.buttons()) != 0:  ## button is pressed; send mouseMoveEvents and mouseDragEvents
+        cur_move_event_time = get_millis()
+        # ignore high frequency events
+        if cur_move_event_time - self._last_move_event_time > 15:
```

Comment:
can you make `15` an attribute of this object?  Seems like this should be a configurable value.  maybe `_milliseconds_between_updates` or something